internal/cli/users_roles.go

```diff
@@ -0,0 +1,65 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/auth0/go-auth0/management"
+	"github.com/spf13/cobra"
+
+	"github.com/auth0/auth0-cli/internal/ansi"
+)
+
+func userRolesCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "roles",
+		Short: "Manage a user's roles",
+		Long: "Manage a user's assigned roles. To learn more about roles and their behavior, read " +
+			"[Role-based Access Control](https://auth0.com/docs/manage-users/access-control/rbac).",
+	}
+
+	cmd.SetUsageTemplate(resourceUsageTemplate())
+	cmd.AddCommand(showUserRolesCmd(cli))
+
+	return cmd
+}
+
+func showUserRolesCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		ID string
+	}
+
+	cmd := &cobra.Command{
+		Use:   "show",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Show a user's roles",
+		Long:  "Display information about an existing user's assigned roles.",
+		Example: `  auth0 users roles show
+  auth0 users roles show <user-id>
+  auth0 users roles show <user-id> --json`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				if err := userID.Ask(cmd, &inputs.ID); err != nil {
+					return err
+				}
+			} else {
+				inputs.ID = args[0]
+			}
+
+			var userRoleList *management.RoleList
+			if err := ansi.Waiting(func() (err error) {
+				userRoleList, err = cli.api.User.Roles(inputs.ID)
```

Comment:
Will this get all roles or just the first 50? Would be nice to enable the assignment of all roles.

Reply:
This will only get 50 roles indeed 👍🏻 Will add pagination as well. But I'd keep it to just 50 tho for the multi select in the add and remove PRs that follow this, because having more than 50 lines of items to select each on a new line is bad DX imo, wdyt?